src/components/TaskTracker/Task.tsx

```diff
@@ -94,9 +117,42 @@ export const Task = ({ task }) => {
             </div>
           </div>
         </div>
+
       ) : (
         <Settings setOpenSettings={setOpenSettings} Task={task} />
       )}
+      <div
+        className="absolute">
+        {task.menuToggled && (
```

Comment:
Like @Hacksore mentioned - each task can potentially open multiple side menu's at once

Reply:
Yeah, I just threw a linear search in there for if the ContextMenu is toggled in any other task, then we just close the other ContextMenu.

Resolved in bb8646408e6fdeb134eae00e4c8d3a254d9c0a81